pkgs/desktops/gnome/extensions/collisions.json

```diff
@@ -101,6 +101,10 @@
       "workspace-indicator@gnome-shell-extensions.gcampax.github.com",
       "horizontal-workspace-indicator@tty2.io"
     ],
+    "lock-keys": [
```

Comment:
I can't comment on unmodified files (GitHub, grrrr), but please move up the respective entry in `extensionRenames.nix`.

Reply:
Thank you for the reminder.